tests/test_qa_h1_seed_data.py

```
@@ -0,0 +1,188 @@
+            for r in db.conn.execute(
+                "SELECT id, commander, controller, firearm_equipment, cannon_equipment "
+                "FROM armies"
+            ).fetchall()
+        }
+        g = rows["guanning"]
+        assert "袁崇焕" not in g["commander"]
+        assert int(g["cannon_equipment"]) >= 4
+        assert int(g["firearm_equipment"]) >= 55
+        d = rows["dongjiang"]
+        assert d["commander"] == "毛文龙"
+        # 人物卡：毛文龙 active；袁崇焕不得仍是 commander
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)